pulp_container/app/registry_api.py

```diff
@@ -547,11 +547,13 @@ class BlobUploads(ContainerRegistryApiMixin, ViewSet):
     model = models.Upload
     queryset = models.Upload.objects.all()
 
-    content_range_pattern = re.compile(r"^(?P<start>\d+)-(?P<end>\d+)$")
+    content_range_pattern = re.compile(r"^([0-9]+)-([0-9]+)$")
 
     def create(self, request, path):
         """
         Create a new upload.
+
+        Note: We do not support monolithic upload.
```

Comment:
Is "monolithic upload" and "all the file in one chunk" a different thing?

Reply:
yes, monolitic upload uploads all the data during PUT request